### DISCUSSION TITLE
Pivot table

### DISCUSSION THREAD

**Discussion by jedi1992** [EvgSkv/logica#116] [Q&A] [answered]
Hi

1) In your covid example example  there was a way to convert columns of a predicate into rows.  (CasesOverTime to Series). 
Any examples for converting rows into columns esp in the context of pivot table.  Some example like this https://towardsdatascience.com/pivot-in-bigquery-4eefde28b3be in logica

2) Also as we work in Juypter workbook any quick way to describe a public table without having to leave the workspace. Something like 

print(SqlExpr("DESCRIBE bigquery-publicdata....." )); 
or its equivalent predicate
print(SqlExpr("DESCRIBE CovidData." ));  

Thanks

**Comment by EvgSkv** [comment, marked as answer]
Hello!

1. There is no native support for `PIVOT` expression, so for now it needs to be done manually or with Python. Here is an example: [Table Pivot](https://colab.research.google.com/github/EvgSkv/logica/blob/main/examples/more/Table_Pivot.ipynb)

2. You can query the schema with Logica: [example](https://colab.research.google.com/github/EvgSkv/logica/blob/dev/examples/more/Table_Inspection.ipynb)

Let me know if you have further questions.
